src/omv/modules/py_tf.c

```diff
@@ -620,11 +531,13 @@ STATIC mp_obj_t py_tf_segment(uint n_args, const mp_obj_t *pos_args, mp_map_t *k
 STATIC MP_DEFINE_CONST_FUN_OBJ_KW(py_tf_segment_obj, 2, py_tf_segment);
 
 STATIC mp_obj_t py_tf_detect(uint n_args, const mp_obj_t *pos_args, mp_map_t *kw_args) {
-    enum { ARG_roi, ARG_thresholds, ARG_invert };
+    enum { ARG_roi, ARG_thresholds, ARG_invert, ARG_input_scale, ARG_input_normalization };
```

Comment:
Should be the same as `py_tf_segement` see the review comments above.

Reply:
This is an outdated comment. I changed it previously to what was discussed in the meeting with mean/std.